pkg/upgrade/machine_creator.go

```diff
@@ -0,0 +1,283 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package upgrade
+
+import (
+	"fmt"
+	"github.com/blang/semver"
```

Comment:
ehhhh i have some imports thing i have to figure out on my new computer. i lost settings (intentionally) and just haven't set this one up yet. plz ignore.